collector/lib/NRadix.cpp

```diff
@@ -73,8 +73,8 @@ bool NRadixTree::Insert(const IPNet& network) const {
       // Reset and move to lower part.
       bit = 0x8000000000000000ULL;
       if (network.bits() >= 64) {
-        *addr_p++;
-        *net_mask_p++;
```

Comment:
These aren't strictly part of this PR but the compiler warnings were annoying so I thought it was worth fixing